src/test/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporterTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jboss.aerogear.keycloak.metrics;
+
+import org.hamcrest.MatcherAssert;
+import org.junit.Test;
+import org.keycloak.events.Event;
+import org.keycloak.events.EventType;
+import org.keycloak.events.admin.OperationType;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class PrometheusExporterTest {
+  private Event createEvent(EventType type) {
+    Event event = new Event();
```

Comment:
I personally would like use the `final` keyword a bit more here (e.g. on the `type` and the `event)

@ziccardi do you agree ?

Reply:
It's a good practice, like defaulting to `const` in javascript. Will keep this in mind.

Reply:
@matzew Absolutely. Making the parameter `final` ensures that the parameter do not get reassigned inside the method (a very bad practice).
The same for the event variable: if it is not meant to be reassigned, it should be final

Reply:
immutability ftw :)